### ISSUE TITLE
Include short descriptions about the contents of each session.

### ISSUE THREAD

**Issue by danschlet** [swissmobidevs/appbuilders16#10]
This time I couldn't find any information about contents of sessions besides the session title, which was only visible on the web site though (and not in the app). Such a description doesn't have to be long, I think every speaker should be able to give you some information in advance what he/she wants to talk about.

**Comment by bontoJR** [comment]
Definitely something to improve, to be honest, we had all the descriptions, but space for information was limited and we had to prioritise the biography. We will do our best to improve this making sure that 1 week before the conference, all the talks have a title and short description.

**Comment by danschlet** [comment]
Thanks for the answer. To be honest, I think you've put too much priority on the biography. It was great to hear these very renowned or even prominent speakers and it's interesting to know the background of a speaker, but I think a tech conference shouldn't be a personality show. Therefore content should have at least the same importance as biography...

**Comment by ast3150** [comment]
I agree with that @danschlet Also I think next time we should also reflect this in the app as well: In the schedule list we should put the title of the talk before the speaker name. Because due to the limited space, a lot of the titles are truncated and then you don't see the topic of the talk in the overview.

**Comment by akosma** [comment]
I agree with @danschlet too. The abstracts of the sessions are much more important.
